### PR TITLE
ISS-34 add slesh to url on the end if they dont have

### PR DESCRIPTION
**Pull request by Hamilkar247**
example of create middleware
https://medium.com/@hassanzadeh.sd/how-to-write-django-middleware-in-10-minutes-cdc4a2e4e08

Signed-off-by: Hamilkar247 <adonnibal96@gmail.com>

### CODE REVIEW

**Comment by Hamilkar247** [comment]
fixes #34

File: bostardecksweb/settings.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'customMiddleware',
```
Comment (jlatusek) [inline bostardecksweb/settings.py:46]: give it proper name

Comment (Hamilkar247) [reply bostardecksweb/settings.py:46]: why this isn't proper?

Comment (jlatusek) [reply bostardecksweb/settings.py:46]: I think that creating middleware this kind isn't proper solution this problem, and creating something called custom is wrong. It says nothing about kind of operation

**Pull request closed without merging**